install/helm/agones/templates/controller.yaml

```diff
@@ -114,10 +120,19 @@ spec:
         - name: certs
           mountPath: /home/agones/certs
           readOnly: true
+{{- if .Values.agones.controller.persistentLogs }}
+        - name: logs
+          mountPath: /home/agones/logs
+          readOnly: false
+{{- end }}
       volumes:
       - name: certs
         secret:
           secretName: {{ template "agones.fullname" . }}-cert
+{{- if .Values.agones.controller.persistentLogs }}
```

Comment:
Curious - why use an emptyDir? Couldn't we write to the local disk regardless?

We don't need to share the volume between containers do we?

Reply:
emptyDir survives container restarts, not a huge deal. I originally had a PVC here which would let you keep long-term logs on a persistent disk if you choose to. Would that be useful?

Reply:
Ah, gotcha. Yes - keeping that around for container restarts is :+1: that makes a lot of sense.